packages/openapi-generator/src/parser/refs.ts

```diff
@@ -72,7 +72,7 @@ export class OpenApiDocumentRefs {
       (mapping, originalName, i) => ({
         ...mapping,
         [`#/components/schemas/${originalName}`]: {
-          schemaName: schemaNames[i],
+          schemaName: pascalCase(options.schemaPrefix ?? '') + schemaNames[i],
```

Comment:
[pp] I think if we give the user the flexibility to assign a prefix, we should leave it as given.